.github/workflows/ci.yml

```
@@ -367,8 +371,9 @@ jobs:
           # __kuser_helper_version < 5, etc., and is not a public API.
           RUSTDOCFLAGS: ${{ env.RUSTDOCFLAGS }} --cfg portable_atomic_test_outline_atomics_detect_false
           RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg portable_atomic_test_outline_atomics_detect_false
-        # powerpc64 is skipped because tested below.
-        if: (matrix.target == '' && !contains(matrix.rust, 'i686') || startsWith(matrix.target, 'x86_64')) || startsWith(matrix.target, 'aarch64') && !(contains(matrix.target, '-musl') && matrix.flags == '') || startsWith(matrix.target, 'armv5te') || matrix.target == 'arm-linux-androideabi'
+        # PowerPC 64-bit is skipped because it is tested below.
```

I'm not sure why the comment needs to be changed here too. In this file we consistently use “powerpc64”; even if we look at the entire repository, we use only “powerpc64” or “PowerPC64”.

---

reverting this update now:

```suggestion
        # powerpc64 is skipped because tested below.
```